lib/juice.js

```diff
@@ -220,10 +220,16 @@ function inlineDocument($, css, options) {
     var elName = el.name.toUpperCase();
     if (juice.widthElements.indexOf(elName) > -1) {
       for (var i in el.styleProps) {
-        if (el.styleProps[i].prop === 'width' && el.styleProps[i].value.match(/px/)) {
-          var pxWidth = el.styleProps[i].value.replace('px', '');
-          $(el).attr('width', pxWidth);
-          return;
+        if (el.styleProps[i].prop === 'width') {
+          if (el.styleProps[i].value.match(/px/)) {
+            var pxWidth = el.styleProps[i].value.replace('px', '');
+            $(el).attr('width', pxWidth);
+            return;
+          }
+          if (juice.tableElements.indexOf(elName) > -1 && el.styleProps[i].value.match(/\%/)) {
```

Comment:
The Spec (https://www.w3.org/wiki/HTML/Elements/img)  states: 

"Give the width of the visual content of the element, in CSS pixels."

I'm not really sure how older clients like outlook would treat images with percentages, that's why I decided to apply it just for table elements.